content/blog/2025/04/2025-04-11-twim.md

```diff
@@ -0,0 +1,351 @@
+> 
+> 🎓 You can find more info about the position and [apply here](<https://www.stillinger.aau.dk/phd-stillinger/vis-stilling/vacancyId/1219575>).
+> 
+> 🎦 This project is directly related to "the walled garden dilemma" I presented at the MatrixConf, and you can [watch the video here](<https://youtu.be/0OWAPYS2QIE?si=CUIo69rAN7LS3cze>).
+
+[Florian Jacob](https://matrix.to/#/@bh7953:kit.edu) announces
+
+> Our research paper [“ReP2P Matrix: Decentralized Relays to Improve Reliability and Performance of Peer-to-Peer Matrix”](https://dl.acm.org/doi/abs/10.1145/3694809.3700741), is now available under CC-BY, [code and data based on the existing peer-to-peer Matrix prototypes](https://gitlab.kit.edu/kit/tm/telematics/rep2p-matrix) is available as well. We thought of a concept for “disposable decentralized relays” - like simple, user-independent homeservers with minimal privileges and without users being bound to only one - and evaluated it as an additional way of message dissemination that complements the current P2P Matrix prototypes. The work was a cooperation between the [Telematics](https://telematics.tm.kit.edu/) and [Decentralized Systems](https://www.dsn.kastel.kit.edu/) research groups, and was presented at the [Decentralization of the Internet Workshop](https://conferences.sigcomm.org/co-next/2024/#!/din) at [ACM CoNext](https://conferences.sigcomm.org/co-next/2024/#!/home).
+
+## Dept of Clients 📱
+
+
+
+### Element X iOS ([website](https://github.com/vector-im/element-x-ios))
+
+A total rewrite of Element-iOS using the Matrix Rust SDK underneath and targeting devices running iOS 16+.
+
+[Doug](https://matrix.to/#/@douge:matrix.org) reports
+
+> This week we released Element X 25.04.3 with a collection of improvements such as
+> 
+> * The option to only hide timeline media in public rooms.
+> * The ability to kick/ban someone directly from the timeline (and add a reason when you do it).
+> * Improvements to hopefully reduce the amount of space use by both the SDK and the app.
+> 
+> On top of this we’ve continued to work on support for Threads, added the ability to enter your PIN with a keyboard and improved the smoothness of the authentication flow when using OIDC.
+
+### Element X Android ([website](https://github.com/vector-im/element-x-android))
+
+Android Matrix messenger application using the Matrix Rust SDK and Jetpack Compose.
+
+[Benoit](https://matrix.to/#/@benoit.marty:matrix.org) says
+
+> We are releasing a new RC 25.04.1 to fix a crash introduced recently that did not get its way to production. Thanks for all people who have sent rageshakes and to YamatoRyou for having created the issue!
+> 
+> Besides that, we're working on improving accessibility, when using a screen reader. Also there have been some changes on the settings to prevent image from invitation to be visible for safety reason.
```

Comment:
```suggestion
> Besides that, we're working on improving accessibility when using a screen reader. Also there have been some changes on the settings to prevent images from invitations to be visible for safety reason.
```